tests/integrationtest/l4_service_test.go

```diff
@@ -962,46 +961,3 @@ func TestInfraSettingChangeMapping(t *testing.T) {
 	TeardownAviInfraSetting(t, settingName2)
 	TearDownTestForSvcLB(t, g)
 }
-
-func TestBootupServiceLBStatusPersistence(t *testing.T) {
```

Comment:
Are we covering this use case somewhere else?

Reply:
yes bootup scenarios are covered in the functional tests.